### DISCUSSION TITLE
[New] La metadescription est pertinente : title et metadescription

### DISCUSSION THREAD

**Discussion by Ferreole** [cnumr/best-practices#95] [Bonnes pratiques] [answered]
Bonne   pratique #5109 
--
4, conseillé | 2, MEO standard
Les meta concernent notamment le title et la metadescription.   Ces informations se paramétrent dans le back-office   du site.    Leur objectif   identifier le contenu de la page,    répondre à l'intention de recherche,    accrocher pour susciter le clic   Soigner son title est important : il se   retrouve en barre d’adresses. C’est lui également qui figure lors du partage   de l’article et en bleu dans les résultats de Google.   La metadescription n’a pas d’incidence sur le positionnement   du site. En revanche, elle se retrouve souvent dans le résultat du moteur de   recherches. Elle favorise donc le clic.

**Comment by tbroyer** [comment]
> Soigner son title est important […] C’est lui également qui figure lors du partage de l’article et en bleu dans les résultats de Google. La metadescription […] se retrouve souvent dans le résultat du moteur de recherches.

Attention, ce n'est pas/plus forcément le cas:
* https://developers.google.com/search/docs/advanced/appearance/title-link#how-title-links-in-google-search-are-created
* https://developers.google.com/search/docs/advanced/appearance/snippet#how-snippet-created

…même si un title et une metadescription "de qualité" aura ainsi plus de chance de se retrouvée utilisée par Google.
(et puis il n'y a pas que Google dans la vie, et je n'ai pas regardé comment DuckDuckGo ou Bing font de leur côté par exemple)

**Comment by Christophe-Cl** [comment]
BP "like" :
- Opquast 3 : https://checklists.opquast.com/fr/assurance-qualite-web/le-code-source-de-chaque-page-contient-une-metadonnee-qui-en-decrit-le-contenu
- Opquast 98 : https://checklists.opquast.com/fr/assurance-qualite-web/le-titre-de-chaque-page-permet-didentifier-son-contenu
- et RGAA

**Reply by Ferreole** [reply]
J'ai pris connaissance des règles Opqast avant d'écrire ces BP. 
L'objectif n'est pas qu'il y ait une metadonnée mais qu'elle soit pertinente 

Le complément avec exemple est : 

Exemple
Title : il comporte l’expression clé et résume l'objectif de la page. 
Title : Yellow Dolphins- Architecture d’information, SEO, stratégie de contenu – conseil et coaching -  Paris, Luxembourg,  Bruxelles, Malaga 

Metadescription : elle résume la page, complète le title tout en incitant à poursuivre la lecture.
Meta : 
Découvrez nos différentes formules de formation à la gestion de projet et à la rédaction web. Formation. Nous accompagnons passionnément les managers, chefs de projet et rédacteurs web! Coaching. Laissez-vous surprendre par nos formules de coaching de vos équipes web... en pleine nature! Animation d'équipes.

Les volumes – longueur et taille en px - du title et de la metadescription s’évaluent en nombre caractères et en largeur de pixels à l’écran.

**Reply by Christophe-Cl** [reply]
OK merci :) Pour moi c'est plus un sujet SEO / Accessibilité / Contenu pur.
Je m'interroge sur l'impact réel en écoconception ?

**Reply by Ferreole** [reply]
Effectivement, cela a sans doute plus trait au contenu. 
J'ai partagé les BP sur le référencement avec mon prisme sobriété éditoriale. A voir dc la pertinence en ce cas !

**Reply by tbroyer** [reply]
cf. https://github.com/cnumr/best-practices/discussions/93#discussioncomment-1550116

Une bonne description peut permettre à l'utilisateur de détecter un faux-positif dans les résultats de recherche, on évite ainsi une visite "pour rien".

**Reply by Christophe-Cl** [reply]
Cépafo :)

**Reply by DocRoms** [reply]
Totalement en accord avec @tbroyer Sur le sujet. Avoir un titre pertinent peut éviter de charger des pages pour rien :) 
C'est d'ailleurs à cause de titre aguicheurs peut pertinents (dits "putaclic") que de nombreux sites arrivent à attirer du monde (sur le court terme) et à multiplier le taux de rebond, en abaissant leur crédibilité... Je suis plutôt en accord avec cet ajout.

Cf https://digitiz.fr/blog/putaclic/

**Reply by Ferreole** [reply]
Carrément d'accord avec Thomas, il explique mieux que moi ;-)

**Comment by Ferreole** [comment]
Soigner tous les axes de pertinence du contenu disponible à l'internaute avant de cliquer pour réduire le temps de recherche de l'internaute : Metadescription, Title, Fil d'Ariane. Ce qui permet de limiter une erreur de navigation et que l'internaute  se retrouve devant une page qui ne répond pas à son besoin.

**Reply by Christophe-Cl** [reply]
Dans le titre de la BP, rester focus sur la pertinence du contenu

**Comment by ACTLEM** [comment, marked as answer]
J'ai créé l'issue pour ajouter la bonne pratique: https://github.com/cnumr/best-practices/issues/253
